src/content/docs/tutorial-app-performance/instrument-your-site.mdx

```diff
@@ -103,9 +104,9 @@ There are two basic ways to install the browser monitoring agent:
     </TabsPages>
   </Tabs>
 
-## Make data-informed changes [#data]
+## Analyze site performance [#data]
 
-Once you've instrumented your site, you can overview your site's performance with different types of data.
+Once you've instrumented your site, you can overview your site's performance with different types of data. Measuring data
```

Comment:
Seems to just be a hanging Measuring data here?

Reply:
ASDJGLASF

Reply:
Frodo smiling emoji.